bears/python/PyImportSortBear.py

```diff
@@ -182,7 +184,18 @@ def run(self, filename, file,
 
         if new_file != tuple(file):
             diff = Diff.from_string_arrays(file, new_file)
+            all_imports = ''
+            for line_nr in sorted(diff._changes):
+                linediff = diff._file[max(line_nr - 1, 0)]
+                list_of_imports = re.split(', +',
+                                           linediff.rstrip())
+                if list_of_imports[0].startswith('import'):
+                    list_of_imports[0] = list_of_imports[0].lstrip('import')
+                    list_of_imports[0].lstrip()
+                    list_of_imports.sort()
+                    string_of_imports = ' '.join(list_of_imports)
+                    all_imports = all_imports + " " + string_of_imports
```

Comment:
you can do this a bit more elegant :)
Make `all_imports` above to a list and change this line to:
```python
all_imports = []
for line_nr in sorted(diff._changes):
    #....
    all_imports.append(string_of_imports)
```